src/Pages/DetailTaxiDriver/Components/DetailComments.js

```diff
@@ -0,0 +1,284 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+
+const DetailComments = props => {
+  const [starValue, SetstarValue] = useState([
+    false,
+    false,
+    false,
+    false,
+    false,
+  ]);
+  const [starGrade, SetstarGrade] = useState(0);
+  const [comment, Setcomment] = useState('');
+  const [commentList, SetcommentList] = useState([]);
+
+  // star value 계산 함수 (index까지 true로 바꿔서 배열 리턴) star버튼을 클릭시 실행
+  const updateStarValue = index => {
+    let booleanStar = [...starValue];
+    for (let i = 0; i < starValue.length; i++) {
+      index >= i ? (booleanStar[i] = true) : (booleanStar[i] = false);
+    }
+    let yellowStar = booleanStar.filter(el => true === el).length;
+    SetstarGrade(yellowStar);
+    SetstarValue(booleanStar);
+  };
+
+  // input에 값이 들어올때마다 값을 넣기 (input이 onchange될때마다 실행)
+  const getCommentValue = e => {
+    Setcomment(e.target.value);
+  };
+
+  // 받아온 comment를 댓글로 구현하기
+  const pushCommentList = () => {
+    if (comment !== '') {
+      SetcommentList(prev => [...prev, comment]);
+      Setcomment('');
+    }
+  };
+
+  // 엔터키 눌렀을때
+  const enterKey = e => {
+    if (e.key === 'Enter') {
+      uploadReviewData();
+    }
+  };
+
+  // 받아온 별점을 별로 그리기
+  const getStar = grade => {
+    const yellowStar = new Array(Math.floor(grade)).fill(
+      <I color="true" size="true" className="fas fa-star" />
+    );
+    const grayStar = new Array(Math.ceil(5 - grade)).fill(
+      <I size="true" className="fas fa-star" />
+    );
+    const allStar = yellowStar.concat(grayStar);
+    return allStar;
+  };
+
+  const uploadReviewData = () => {
+    const loggedInfo = localStorage.getItem('AUTHORIZATION');
+    fetch(`http://3.34.199.216:8000/taxis/taxidrivers/${props.id}`, {
+      method: 'POST',
+      headers: {
+        AUTHORIZATION: loggedInfo,
+      },
```

Comment:
```suggestion
      headers: {
        AUTHORIZATION: loggedInfo,
      },
```
AUTHORIZATION 전부 대문자로 쓰기로 한건가요?

Reply:
넵 대문자로 쓰기로 했습니다!